### ISSUE TITLE
Filter out 2018

### ISSUE THREAD

**Issue by curran** [cfinst/cfinst.github.io#210]
It seems that a recent data change from @bglavin introduced values for 2018, yielding the following result:
![image](https://cloud.githubusercontent.com/assets/68416/24792647/897ab9a6-1b9c-11e7-8678-a0bde9e91af6.png)

To address this, we need to account for 2018, and filter out its values like we are already doing for 2016.

**Comment by curran** [comment]
@seemantk Just was researching this a bit and thought I'd share my findings:

```
$ ag filterYear
_config.yml
22:filterYear: 2016

js/schematic.js
143:          .map(dataset{% if site.filterYear %}
145:              .filter(function(d) { return d.Year != +{{ site.filterYear }}; })
```

Probably we can make this into an array of years, maybe calling it `filterYears`, and have it say `"2016, 2018"` or whatever is the more canonical "Jekyll way" of representing arrays.

**Comment by seemantk** [comment]
@curran, nice catch and great solution for it, as well.  (As for Jekyll arrays, checkout "authors" in the _config.yml file to see an example.)

What do you think of moving the "filterYears" setting into a file in _data instead?  I'm thinking that that way _config has nothing to do with the dataset, and _data has everything to do with it.

**Comment by seemantk** [comment]
Closed by #213
